packages/web/src/components/molecules/Console/Console.tsx

```diff
@@ -49,6 +49,8 @@ export const Console = forwardRef<ConsoleRef, ConsoleProps>(({content, wrap, Lin
   const containerRef = useRef<HTMLDivElement | null>(null);
   const maxDigits = processor.maxDigits;
 
+  const {execDetailsTab} = useParams();
```

Comment:
Do we actually need that? It would be best if we won't check for params from the console

Reply:
It seems like the logs are not being rendered when switching between tabs... so I used the active tab from params just to rerender when switching back to the logs output

Reply:
Could we base on i.e. visibility instead? It would be more flexible

Reply:
Pushed change